static/app/components/events/eventReplay/replayClipPreview.tsx

```diff
@@ -46,18 +50,31 @@ import FluidHeight from 'sentry/views/replays/detail/layout/fluidHeight';
 import {ReplayCell} from 'sentry/views/replays/replayTable/tableCell';
 import type {ReplayRecord} from 'sentry/views/replays/types';
 
+type AdditionalProps =
+  | {
+      group: Group;
+      clipOffsets?: undefined;
+      eventTimestampMs?: undefined;
+    }
+  | {
+      clipOffsets: {
+        durationAfterMs: number;
+        durationBeforeMs: number;
+      };
+      eventTimestampMs: number;
+      group?: undefined;
+    };
```

Comment:
@michellewzhang so I intentionally did _not_ do that because I wanted to make sure that either the group is provided OR the timestamp information. And we don't want to mix and match passing the group and the timing info because it would be unclear how to derive the timing information.

Reply:
makes sense, thanks for elaborating